src/test/java/teammates/test/cases/browsertests/BaseUiTestCase.java

```diff
@@ -29,7 +29,7 @@
 public abstract class BaseUiTestCase extends BaseTestCaseWithBackDoorApiAccess {
 
     /** Indicates if the test-run is to use GodMode. */
-    private static boolean isGodModeEnabled;
+    private static boolean isGodModeEnabled = true;
```

Comment:
@whipermr5 
@VamsiSangam 
I get 5 errors in Ci Tests. Should I post it here or open a new issue?